### ISSUE TITLE
Show tickbox in Settings - Preferences - Document List Panel is not persisted between sessions

### ISSUE THREAD

**Issue by hickc** [notepad-plus-plus/notepad-plus-plus#9978]
### Description of the Issue

Show tickbox in Settings - Preferences - Document List Panel is not persisted between sessions
This has been the case for a number of recent versions of NotePad++ (e.g. version 8, 7.9.5, 7.9.4).

### Steps to Reproduce the Issue

1. In NotePad++ set Show tickbox in Settings - Preferences - Document List Panel to ticked.
2. Close NotePad++
3. Open NotePad++, 

### Expected Behavior

The Show tickbox in Settings - Preferences - Document List Panel should be ticked after closing and opening NotePad++

### Actual Behavior

The Show tickbox in Settings - Preferences - Document List Panel is cleared

**Comment by sasumner** [comment]
No problems for me in retaining this setting in 8.0 / 7.9.5 / 7.9.4.

**Comment by hickc** [comment]
After further investigation the issue only occurs for me if -nosession command line option is used.
I have verified that this issue occurs with the latest NotePad++ V8.1 running on Windows 10 Pro.
I installed NotePad++ to a user specified installation folder C:\p\npp
Within Settings - Preferences - Document List Panel , if "Show" is ticked, it gets cleared if NotePad++ is started with -nosession command line option.
This does not happen for "Disable extension column", the other tickbox in "Document List Panel".
